yente/search/indexer.py

```diff
@@ -205,11 +197,18 @@ async def update_index(force: bool = False) -> bool:
         catalog = await get_catalog()
         log.info("Index update check")
         changed = False
+        exceptions = []
         for dataset in catalog.datasets:
-            _changed = await index_entities_rate_limit(provider, dataset, force)
-            changed = changed or _changed
-
+            try:
+                _changed = await index_entities_rate_limit(provider, dataset, force)
```

Comment:
@pudo  Untested, but if we want to return an error from the process and still update each index possible then maybe something like this? If there's an error in an index the information about whether there was a change will be lost, but that might be OK if we don't need that level of granulatiry.

Reply:
I wonder if we should have this return a named tuple that contains: `failed`/`changed` as two different return values -